commons/connector/fitbit/fitbit_sleep_classic.avsc

```diff
@@ -9,6 +9,7 @@
     { "name": "duration", "type": "int", "doc": "Duration at this sleep characteristic in seconds." },
     { "name": "level", "type":
       { "name": "FitbitSleepClassicLevel", "type": "enum", "symbols": ["AWAKE", "RESTLESS", "ASLEEP", "UNKNOWN"], "doc": "Level of sleep as computed by Fitbit."},
-      "doc": "Level of sleep as computed by Fitbit.", "default": "UNKNOWN" }
+      "doc": "Level of sleep as computed by Fitbit.", "default": "UNKNOWN" },
+    { "name": "efficiency", "type": "string", "doc": "Calculated sleep efficiency score" }
```

Comment:
Should be nullable, as above. Ideally, the efficiency is a number (int / float) and the doc string explains how it is calculated.

Reply:
Thanks for pointing this out. I have changed the type to `int` and added null as a default value.